docs/developers/documentation/index.md

```diff
@@ -6,27 +6,66 @@ documentation.
 
 ## Organization of the documentation
 
-The napari documentation is built from sources located at the
-[napari/docs](https://github.com/napari/docs) repository on GitHub. That
-repository is where all the narrative documentation (e.g. tutorials, how-to
-guides) pull requests should be made. Meanwhile, changes to docstrings or to the
+The napari documentation is built from multiple sources that are organised into multiple repositories:
+1. The main napari documentation is built from sources located at the
+[napari/docs](https://github.com/napari/docs) repository on GitHub.
+That repository is where all the narrative documentation (e.g. tutorials, how-to
+guides) pull requests should be made.
+This narrative napari documentation is written in
+[Myst markdown](https://myst-parser.readthedocs.io/en/latest/index.html),
+a version of [commonmark](https://spec.commonmark.org/) Markdown
+(see a [markdown cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet))
+with some additional features.
+2. Some of the documentation about [plugins](https://napari.org/plugins) resides in
+the [napari/npe2](https://github.com/napari/npe2) repository
+(e.g. the [contributions reference page](https://napari.org/plugins/contributions.html)) and should be modified there.
+This is documentation is written with [Jinja](https://jinja.palletsprojects.com/en/3.1.x/).
+3. Changes to docstrings (for the [API reference](https://napari.org/stable/api/index.html)) or to the
 [examples gallery](https://napari.org/gallery) should be made to the
 [napari/napari](https://github.com/napari/napari) repository (see also
-[](add-examples)).
+[the guide on adding examples to the gallery](add-examples)).
+4. Changes to how [napari.org](https://napari.org/dev) appears site-wide are made to the
+[napari-sphinx-theme](https://github.com/napari/napari-sphinx-theme).
 
-## Prerequisites
+(contributing-without-local-setup)=
+## Contributing to the napari documentation without a local setup
+
+If you would like to see new documentation added to napari or want to see changes to existing documentation, but are not clear on these instructions or don't have the time to write it yourself, you can [open an issue](https://github.com/napari/docs/issues/new/choose).
+If you are adding new documentation or modifying existing documentation and would prefer a simpler workflow than the local setup guide described below,
+you can use the GitHub web interface to open your pull request
+by either [uploading file(s) from your computer](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository),
+[creating and editing a new file](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files)
+or [editing an existing file](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files)
+on the [napari-docs](https://github.com/napari/docs) GitHub repository.
```

Comment:
I love this, can we also add a note about doing this from your fork through a feature branch?

Reply:
Committed to add - "You can also first [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [napari-docs](https://github.com/napari/docs) repository to your own GitHub account, create a [feature branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch), upload/create/edit files through the GitHub web interface, and then [open a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) back to [napari-docs](https://github.com/napari/docs)."